v4/test/classifications/classification_with_where_test.go

```diff
@@ -0,0 +1,97 @@
+package classifications
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/semi-technologies/weaviate-go-client/v4/test/testsuit"
+	"github.com/semi-technologies/weaviate-go-client/v4/weaviate/classifications"
+	"github.com/semi-technologies/weaviate-go-client/v4/weaviate/filters"
+	"github.com/semi-technologies/weaviate-go-client/v4/weaviate/testenv"
+	"github.com/semi-technologies/weaviate/usecases/classification"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClassifications_With_Where_Filters_integration(t *testing.T) {
+
+	t.Run("up", func(t *testing.T) {
+		err := testenv.SetupLocalWeaviate()
+		if err != nil {
+			fmt.Printf(err.Error())
+			t.Fail()
+		}
+	})
+
+	t.Run("POST /classifications", func(t *testing.T) {
+		client := testsuit.CreateTestClient()
+		createClassificationClasses(t, client)
+
+		sourceWhere := filters.Where().
+			WithOperator(filters.Like).
+			WithPath([]string{"id"}).
+			WithValueString("*")
+		classifyProperties := []string{"taged"}
```

Comment:
`taged` -> maybe meant to be `tagged`?

Reply:
Actually that's the name of property tak we use in our tests. I have just re-used that.
Let me check if it would be a quick to fix that property name.

Reply:
OK. I have fixed all of the occurrences of this property